README.md

```diff
@@ -18,6 +18,15 @@ The master branch is in version 1.5.0-SNAPSHOT:
 
 ### Dependency Information
 
+
+
+As of 1.5.0, Alluxio provides several different client interfaces. The Alluxio file system interface
+provided by `alluxio-core-client-fs` is suggested for performance and flexibility. If you want to
```

Comment:
replace "suggested for performance and flexibility" with "recommended for best performance and access to Alluxio-specific functionality"